packages/fast-usdc/test/exos/settler.test.ts

```diff
@@ -280,6 +285,11 @@ test('slow path: forward to EUD; remove pending tx', async t => {
         value: 'osmo183dejcnmkka5dzcu9xw6mywq0p2m5peks28men',
       },
       usdc.units(150),
+      {
+        forwardOpts: {
+          intermediateRecipient: 'noble1test',
```

Comment:
Using the same property name for a string here and a `ChainAddress` elsewhere is hurting my tiny brain.

Reply:
Agree, in hindsight this was a poor decision. Updated everything to use the `ChainAddress` object.